### DISCUSSION TITLE
HikVision Camera motion detection

### DISCUSSION THREAD

**Discussion by Kabre555** [Sunoo/homebridge-camera-ffmpeg#1119] [Q&A]
I have a hikvision camera setup with motion/ring doorbell.. (all options under automations) are enabled.   The buttons are created, but not triggered automatically when motion is sensed.    Am I missing something, I do have it working with Home Assistant and it sees the trigger alerts for motion, but not via HomeBridge.

I would like to have the notifications pop up on my apple tv

this works if I manually click the buttons under accessories, but its not working with this camera under this setup unless I am missing something.

**Comment by Sunoo** [comment]
Unless your camera supports MQTT or making an HTTP request on motion, it won’t work directly with this plugin. I have additional plugins that add support for FTP and SMTP based alerts (most cameras support at least one).

As I do not have a HikVision camera, I have not written support for “native” motion detection directly, if it has any, but one of the other methods should work just as well.

https://sunoo.github.io/homebridge-camera-ffmpeg/automation/

**Reply by Kabre555** [reply]
under the HomeAssistant plugin it requires  'Notify Surveillance Centre' to be checked, so I guess it uses a different method to link up.  I do have the ability to upload an image to FTP on the camera when motion is detected so I can probably that route.   Is there a way to manually trigger the Accessory Motion button from HomeAssistant into Homebridge.?     can I please check out your FTP plugin .

thanks

**Reply by Kabre555** [reply]
never mind found the plugin,  will I be able to trigger an AppleTV notification using this method?

**Reply by Sunoo** [reply]
To trigger AppleTV with motion alerts, you’ll need to set motionDoorbell to true in the config for this plugin. When that’s set, any motion should appear on the Apple TV.

**Comment by Kabre555** [comment]
ok , so thats enabled.. but I think I need the smtp plugin, so walk me through the logic of this..  this create a homebridge smtp server and I configure the camera to send email to the homebridge IP with any username/pw combination and that will convert to a http trigger that will set off the MotionDoorBell accessory ?

**Reply by Sunoo** [reply]
That’s basically correct, yes. The email address you send to needs to be the camera name @ any domain.

**Reply by Kabre555** [reply]
ok so the smtp addon is not working, my camera test messages are failing to communicate with the server.   Also I ran a port scan on the ip of the homebridge machine and the smtp port is not showing as open.    Is there any additional settings required other then just installing the Homebridge SMTP Motion

**Reply by Kabre555** [reply]
ok.   got it to work using Raspi version, the synology docker and virtual machine image didn't work.

**Reply by Sunoo** [reply]
I wish I knew more about Docker, sorry. At least you got it going somehow.

**Comment by Kabre555** [comment]
wow, so close and now a road block..  I'm using the Docker version under synology and I can't add the 2525 port and I think the camera email is not reaching the homeassistant  :(

**Reply by Sunoo** [reply]
I’m not familiar with Docker or Synology, but you can set smtp_port to anything you want, if another port works better.
